src/main/java/hw6/pages/MetalAndColorsPage.java

```diff
@@ -0,0 +1,214 @@
+package hw6.pages;
+
+import com.epam.jdi.light.elements.complex.Checklist;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.XPath;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Checkbox;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+
+public class MetalAndColorsPage extends WebPage {
+
+    // Colors dropdown
+    @JDropdown(root = "div[ui=dropdown]",
+            value = ".filter-option",
+            list = "li",
+            expand = ".caret")
+    public Dropdown colorsDropdown;
+
+    public void selectColor(String colorValue) {
+        colorsDropdown.select(colorValue);
+    }
+
+    //Metals dropdown
+    @JDropdown(
+            root = "div[ui=combobox]",
+            value = "input",
+            list = "li",
+            expand = ".caret"
+    )
+    public Dropdown metalsDropdown;
+
+    public void selectMetal(String metalValue) {
+        metalsDropdown.select(metalValue);
+    }
+
+    // Numbers summary
+    @UI("[name=custom_radio_odd]")
+    public RadioButtons oddNumbers;
+
+    @UI("[name=custom_radio_even]")
+    public RadioButtons evenNumbers;
+
+    public void selectNumbers(int[] numbers) {
+        int first;
+        int second;
+        switch (numbers[0]) {
+            case (1):
+                first = 1;
+                break;
+            case (3):
+                first = 2;
+                break;
+            case (5):
+                first = 3;
+                break;
+            default:
+                first = 4;
+                break;
+        }
+        switch (numbers[1]) {
+            case (2):
+                second = 1;
+                break;
+            case (4):
+                second = 2;
+                break;
+            case (6):
+                second = 3;
+                break;
+            default:
+                second = 4;
+                break;
+        }
+
+        oddNumbers.select(first);
+        evenNumbers.select(second);
+    }
+
+
+    //Elements checklist
+    @FindBy(css = "section[id=elements-checklist] input[type=checkbox]")
+    public Checklist earthElements;
+
+    @XPath("//input[@id='g1']")
```

Comment:
You have a earthElements Checklist, so what is the reason of checkboxes fields below ?
You need to get rid of them.